modules/claims_api/lib/bgs_service/vnp_person_service.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VnpPersonService < ClaimsApi::LocalBGS
+    # Takes an object with a minimum of (other fields are camelized and passed to BGS):
+    # procId: BGS procID
+    # ptcpntId: Veteran's participant id
+    # firstNm: Veteran's first name
+    # lastNm: Veteran's last name
+    #
+    # The target veteran can be passed in as an optional second argument
+    # this overrides the ptcpntId, firstNm, and lastNm fields in the opts.
+    #
+    # The icn can be passed in as an optional third argument
+    # this does an MPI lookup to override the ptcpntId, firstNm, and lastNm fields in the opts.
+    def vnp_person_create(opts, target_veteran: nil, icn: nil)
+      opts = opts.dup
+      opts = icn_opts(icn).merge(opts) if icn
+      opts = target_veteran_opts(target_veteran).merge(opts) if target_veteran
+      opts.transform_keys! { |k| k.to_s.camelize(:lower) }
+
+      validate_opts! opts
+
+      opts = jrn.merge(opts)
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <arg0></arg0>
+      EOXML
+
+      opts.each do |k, v|
+        node = Nokogiri::XML::Node.new k.to_s, body
+        node.content = v.to_s
+        node.set_attribute('xsi:nil', 'true') if v.blank?
+        opt = body.at('arg0')
+        node.parent = opt
+      end
+
+      make_request(endpoint: 'VnpPersonWebServiceBean/VnpPersonService', action: 'vnpPersonCreate', body:,
+                   key: 'return')
+    end
+
+    private
+
+    def jrn
+      {
+        jrnDt: Time.current.iso8601,
+        jrnLctnId: Settings.bgs.client_station_id,
+        jrnStatusTypeCd: 'U',
+        jrnUserId: Settings.bgs.client_username,
+        jrnObjId: Settings.bgs.application
+      }
+    end
+
```

Comment:
@acovrig I don't think we need to pass in the icn or target_veteran if we are already passing `vnpPtcpntId: target_veteran.participant_id,` as an arg.